### PR TITLE
Stop thread pool before quitting scheduler

### PR DESCRIPTION
**Pull request by huachaohuang**
Quitting scheduler without stopping thread pool can cause potentially coredump.

### CODE REVIEW

File: src/storage/txn/scheduler.rs [hunk 1]
```diff
@@ -1432,6 +1432,16 @@ impl Scheduler {
             }
         }
     }
+
+    fn quit(&mut self) -> Result<()> {
```
Comment (siddontang) [inline src/storage/txn/scheduler.rs]: on_quit

**Comment by ngaut** [comment]
LGTM

**Comment by huachaohuang** [comment]
PTAL @BusyJay

File: src/storage/txn/scheduler.rs [hunk 2]
```diff
@@ -1432,6 +1432,16 @@ impl Scheduler {
             }
         }
     }
+
+    fn on_quit(&mut self) -> Result<()> {
```
Comment (BusyJay) [inline src/storage/txn/scheduler.rs]: I think `shutdown` is better.

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**